src/modules/keyboardmanager/ui/ShortcutControl.h

```diff
@@ -25,6 +28,11 @@ class ShortcutControl
 
     ShortcutControl(const int& rowIndex, const int& colIndex)
     {
+        shortcutDropDownStackPanel.Background(Windows::UI::Xaml::Media::SolidColorBrush{ Windows::UI::Colors::LightGray() });

```

Comment:
We probably should use `shortcutDropDownStackPanel.RequestedTheme(ElementTheme::Light);`, so that we get the correct colors automatically.